### PR TITLE
Fix PR #81

### PR DESCRIPTION
**Pull request by tig**
Fixes Pull request #81 

https://github.com/PowerShell/GraphicalTools/pull/81

### CODE REVIEW

File: src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs [hunk 1]
```diff
@@ -27,10 +27,11 @@ public HashSet<int> Start(ApplicationData applicationData)
...
             _gridViewDetails = new GridViewDetails
             {
-                // Have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
-                ListViewOffset = 8
+                // If we have an OutputMode, then we want to make them selectable. If we make them selectable,
+                // they have a 8 character addition of a checkbox ("     [ ]") that we have to factor in.
+                ListViewOffset = _applicationData.OutputMode != OutputModeOption.None ? 8 : 4
             };
-
+
```
Comment (iSazonov) [inline src/Microsoft.PowerShell.ConsoleGuiTools/ConsoleGui.cs:34]: Please revert the change.

You need to turn on a feature to remove trailing spaces in your editor.

**Comment by TylerLeonhardt** [comment]
Do we need this PR anymore?

**Comment by tig** [comment]
It appears we do not. I can't tell how my fixes to the PR that had issues got merged in, but they are there...
